### PR TITLE
NAS-110024 / 21.08 / lagg name bond0

### PR DESCRIPTION
**Pull request by AlexKarpov98**

### CODE REVIEW

**Review by undsoft — COMMENTED** [review]
It looks good for me, but let's wait for an approval from someone more familiar with this.

**Review by william-gr — APPROVED** [review]

**Review by undsoft — APPROVED** [review]

**Pull request merged**
